bw2data/data_store.py

```diff
@@ -162,14 +162,14 @@ class ProcessedDataStore(DataStore):
     dtype_fields = None
     # Numpy columns names can't be unicode
     base_uncertainty_fields = [
-        (numpy_string('uncertainty_type'), np.uint8),
-        (numpy_string('amount'), np.float32),
-        (numpy_string('loc'), np.float32),
-        (numpy_string('scale'), np.float32),
-        (numpy_string('shape'), np.float32),
-        (numpy_string('minimum'), np.float32),
-        (numpy_string('maximum'), np.float32),
-        (numpy_string('negative'), np.bool),
+        (numpy_string('uncertainty_type'), int),
+        (numpy_string('amount'), float),
+        (numpy_string('loc'), float),
+        (numpy_string('scale'), float),
+        (numpy_string('shape'), float),
+        (numpy_string('minimum'), float),
+        (numpy_string('maximum'), float),
+        (numpy_string('negative'), bool),
```

Comment:
Changing `np.bool` to `bool` is perfect, but please revert the other changes, I don't want 64 bit floats for the other values.